website/src/components/Messages/MessageTableEntry.tsx

```diff
@@ -158,6 +168,25 @@ export function MessageTableEntry({
           <LabelMessagePopup message={message} show={labelPopupOpen} onClose={closeLabelPopup} />
           <ReportPopup messageId={message.id} show={reportPopupOpen} onClose={closeReportPopup} />
         </HStack>
+        <Flex position="absolute" gap="2" top="-2.5" right="5">
+          {showAuthorBadge && message.user_is_author && (
+            <Tooltip label={t("message_author_explain")} placement="top">
+              <Badge size="sm" colorScheme="green" textTransform="capitalize">
+                {t("message_author")}
+              </Badge>
+            </Tooltip>
+          )}
+          {message.deleted && isAdminOrMod && (
+            <Badge colorScheme="red" textTransform="capitalize">
+              Deleted
```

Comment:
should this also be moved to localization? If not, add a comment to indicate why.

Reply:
It's admin only feature

Reply:
great, please add comment in the code, so no one tries to extract it out to the locale.